pkg/inngest/batch.go

```diff
@@ -47,6 +47,8 @@ func NewEventBatchConfig(conf map[string]any) (*EventBatchConfig, error) {
 // - The batch is full
 // - The time to wait is up
 type EventBatchConfig struct {
+	Key *string `json:"key,omitempty"`
```

Comment:
there's a `IsValid` function right below.
https://github.com/inngest/inngest/blob/16f8e28ddcb6eca3b28df3319b45f36a7ba363c6/pkg/inngest/batch.go#L64-L83

good to add a simple check to make sure the CEL expression itself is valid, so we can surface issues early when users try to sync the function.

Reply:
Added, thanks for the pointer!